css/components/btn.postcss

```diff
@@ -75,10 +71,10 @@
 /* ICONS */
 
 .btn__icon {
-    @apply w-[1.4em] h-full;
-    @apply stroke-[0.3px] md:stroke-[0.35px] lg:stroke-[0.4px];
-    @apply stroke-current;
-  }
+  @apply w-[1.4em] h-full;
+  @apply stroke-[0.3px] md:stroke-[0.35px] lg:stroke-[0.4px];
```

Comment:
sub pixel? what?

Reply:
I dont know. I didnt changed anything in this file my editor just formatted it